.github/CODEOWNERS

```diff
@@ -1,8 +1,6 @@
 # Before changing this file, make sure you have read how we use it:
 # https://www.notion.so/metabase/Pull-Request-Code-Review-Guide-374f05df889748b69d67af753f9bc9b8?pvs=4#36550c04ceab40b5a8b8bee6264b090c
 
-enterprise/frontend/src/embedding-sdk @metabase/embedding
```

Comment:
do you want to delete all of the mentions?

there are 2 more dirs down there.

https://github.com/metabase/metabase/blob/a260601156f5f79f4a0e71f74a5912264b1ea43d/.github/CODEOWNERS#L130-L131